service/controller/key/control_plane.go

```diff
@@ -70,6 +70,10 @@ func ControlPlaneLaunchTemplateName(getter LabelsGetter, masterID int) string {
 	return fmt.Sprintf("%s-master%d-launch-template", ClusterID(getter), masterID)
 }
 
+func ControlPlaneMasterIgnitionPath(path string, masterID int) string {
+	return fmt.Sprintf("%s%d", path, masterID)
+}
```

Comment:
@calvix I am not sure about this in https://github.com/giantswarm/aws-operator/pull/2368. Why does that have to be separate? Should I add it to the other PR? Should I? Should we drop this?

Reply:
no, you sorted the ignition paths different way, this is not needed